set.mm

```diff
@@ -531826,6 +531823,21 @@ to being a member of a class abstraction (that every class is equal to a
                 ( A e. B <-> ( A e. _V /\ ph ) ) ) $=
     ( wcel cvv elex biadani ) BCDBEDABCFG $.
 
+  $( A set is a binary relation if and only if it belongs to the powerclass of
```

Comment:
Is it "powerclass" or "power class"? We have  "powerclass" 13 times and "power class" 85 times in set.mm.

Reply:
If we want to uniformize, then I'd vote for "powerclass", because it's one word for one concept.